### DISCUSSION TITLE
How to get the center of a popup?

### DISCUSSION THREAD

**Discussion by jason-symbiote** [uber/react-map-gl#1376] [Q&A]
Hi,
Thanks for this great plugin, It handles our map requirements in a nice way:)

I am building a map with markers and when user clicks on a marker it popups with some information and map flies to the centre of the popup and active marker.
I wonder if I can get the centre of a popup's? Because when I render a popup I simply use the lat and lon of the marker and make the popup's anchor props 'bottom'. How do I calculate the centre of a popup?

**Comment by macobo** [comment]
Recently solved a similar problem with markers and popups - in our case, we just panned to the position of the marker, which you would know when you're rendering the popup.
